docs/docs/explanations/standard_library/traits.md

```diff
@@ -0,0 +1,140 @@
+---
```

Comment:
Hey sorry I'm a bit late for this PR, wondering if it would be worth to have some examples @jfecher ?

Reply:
Issue created: https://github.com/noir-lang/noir/issues/3969